DataProcessing/datax-host/src/main/scala/datax/fs/HadoopClient.scala

```diff
@@ -293,6 +302,7 @@ object HadoopClient {
     */
   @throws[IOException]
   def writeHdfsFile(hdfsPath: String, content: String, overwriteIfExists:Boolean) {
+    resolveStorageAccountKeyForPath(hdfsPath)
```

Comment:
This call to resolve key here looks redundant because the writeHdfsFile called inside here again calls resolve. Lets remove this line.

Reply:
writeHdfsFile method does not call resolveStorageAccountKeyForPath anymore. It has been moved out from writeHdfsFile(hdfsPath: String, content: Array[Byte], conf: Configuration, overwriteIfExists:Boolean) method